Programm started at: 2025-10-20 02:03:22

Running in version 1.2[2025-10-20 11:08:34]
Error 1 at reading or storing sensor data.
	I2C read failed

	Traceback (most recent call last):
  File "/home/raspi/CO2_Project/main.py", line 211, in <module>
    g_co2_sensor.read_all()
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 192, in read_all
    self.read_logorithm_results()
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 188, in read_logorithm_results
    raise Exception (pigpio.error_text(b))
Exception: I2C read failed


[2025-10-20 23:32:56]
Git repositories synchronized
	

	NoneType: None


[2025-10-20 23:32:56]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-10-21 02:02:44

Running in version 1.2

[2025-10-21 08:34:54]
Error 1 at reading or storing sensor data.
	I2C read failed

	Traceback (most recent call last):
  File "/home/raspi/CO2_Project/main.py", line 211, in <module>
    g_co2_sensor.read_all()
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 192, in read_all
    self.read_logorithm_results()
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 188, in read_logorithm_results
    raise Exception (pigpio.error_text(b))
Exception: I2C read failed


[2025-10-21 10:55:58]
Error 1 at reading or storing sensor data.
	I2C read failed

	Traceback (most recent call last):
  File "/home/raspi/CO2_Project/main.py", line 211, in <module>
    g_co2_sensor.read_all()
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 192, in read_all
    self.read_logorithm_results()
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 188, in read_logorithm_results
    raise Exception (pigpio.error_text(b))
Exception: I2C read failed


